facade/src/main/java/com/redhat/lightblue/migrator/facade/DAOFacadeBase.java

```diff
@@ -208,6 +208,15 @@ public T call() throws Exception {
         }
     }
 
+    private Throwable extractUnderlyingException(Throwable t) {
```

Comment:
Apache Commons Lang is already in the classpath, so you could use ExceptionUtils.getRootCause(throwable) instead this. I think it also traverses the exception chain to find the last one.